mantis-runtime/src/main/java/io/mantisrx/runtime/command/LoadValidateCreateDir.java

```diff
@@ -96,7 +88,7 @@ public void execute() throws CommandException {
                     File jobDescriptor = new File(fileLoop.getParent() + "/" + jsonFile);
                     new CreateJobDescriptorFile(job, jobDescriptor, fileVersion, fileBase).execute();
                 } catch (Exception e) {
-                    System.out.println("Got an error " + e.toString());
+                    System.out.println("Got an error " + e.getMessage());
```

Comment:
Get rid of System.out.println. Use log.info instead.

Reply:
Should I use Logger instead of System.out.println inside mantis runtime module in all places used?

Reply:
****
<img width="791" alt="Screenshot 2023-06-29 at 10 30 26 AM" src="https://github.com/Netflix/mantis/assets/16557220/5e879205-f8bd-4e7b-84e8-842862113e27">

Reply:
Yes, let's try to eliminate System::out usages in the runtime module. Let's do that in a separate PR though. This PR is big enough already.

Reply:
Added the logger.